app/steps/ui/executors/number/index.js

```diff
@@ -12,7 +12,7 @@ class ExecutorsNumber extends ValidationStep {
 
     getContextData(req) {
         let ctx = super.getContextData(req);
-        ctx.executorsNumber = ctx.executorsNumber ? parseInt(ctx.executorsNumber) : ctx.executorsNumber;
+        ctx.executorsNumber = ctx.executorsNumber ? parseFloat(ctx.executorsNumber) : ctx.executorsNumber;
```

Comment:
what is the use case for using floating point number parsing here rather than integer parsing?

Reply:
Previously it would take the integer in i.e. someone enters 3.912, it would just go through as 3 but not tell the user, so made it parse as float so it would flag it with the correct error message